universe/Effect.h

```diff
@@ -907,11 +907,20 @@ class FO_COMMON_API Effect::GenerateSitRepMessage : public Effect::EffectBase {
     GenerateSitRepMessage(const std::string& message_string, const std::string& icon,
                           const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*> >& message_parameters,
                           ValueRef::ValueRefBase<int>* recipient_empire_id,
-                          EmpireAffiliationType affiliation);
+                          EmpireAffiliationType affiliation,
+                          const std::string label = "",
+                          bool no_stringtable_lookup = false);
```

Comment:
flags should probably be positive indicators of what to do, not negations or indicators of what no to do, especially with the default value false...